drivers/led/lp5562.c

```diff
@@ -481,7 +481,16 @@ static inline int lp5562_set_engine_exec_state(struct device *dev,
 					enum lp5562_led_sources engine,
 					enum lp5562_engine_exec_states state)
 {
-	return lp5562_set_engine_reg(dev, engine, LP5562_ENABLE, state);
+	int ret;
+
+	ret = lp5562_set_engine_reg(dev, engine, LP5562_ENABLE, state);
+
+	/* Delay between consecutive I2C writes to
```

Comment:
Nitpick: Please use the below format for multi line comments:

/*
 * xxx
 */